src/main/com/yetanalytics/lrs/util.cljc

```diff
@@ -14,3 +14,23 @@
        :cljs (.stringify js/JSON (clj->js x) nil 2))
     #?(:clj (json/generate-string x)
        :cljs (.stringify js/JSON (clj->js x)))))
+
+;; a la https://stackoverflow.com/questions/18735665/how-can-i-get-the-positions-of-regex-matches-in-clojurescript
+#?(:cljs
+   (defn regex-modifiers
+     "Returns the modifiers of a regex, concatenated as a string."
+     [re]
+     (str (if (.-multiline re) "m")
+          (if (.-ignoreCase re) "i"))))
+
+#?(:cljs
+   (defn re-pos
```

Comment:
I feel like `re-pos` should have a longer name, like `re-match-with-pos` or similar, since I had trouble figuring out what it actually did when looking at its function calls.

Reply:
I kept the name the same as it is literally copy-pasted from the SO post (in the comment).